enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1546,7 +1546,7 @@ func (ws *workspace) fetch(ctx context.Context, remoteURL string, branches []str
 	if _, err := git(ctx, io.Discard, "remote", "add", remoteName, authURL); err != nil && !isRemoteAlreadyExists(err) {
 		return status.UnknownErrorf("Command `git remote add %q <url>` failed.", remoteName)
 	}
-	fetchArgs := append([]string{"fetch", "--filter=blob:none", "--force", remoteName}, branches...)
+	fetchArgs := append([]string{"fetch", "--config=credential.helper=", "--filter=blob:none", "--force", remoteName}, branches...)
```

Comment:
I think it needs to be `git --config=credential.helper= fetch` instead of `git fetch --config=credential.helper=`

`--config / -c` is a global git flag not specific to a git subcommand. We might change that as well in `git-clone` if needed.

Reply:
ah thank you, I was a little perplexed why this didn't work :)

Reply:
seems `fetch` only supports the short form (-c`) - `git --config=credential.helper=` didn't work

Reply:
The `clone` command is working fine, so will leave that as-is.